mantidproject/mantid
# Fix Lambda min and Lambda max in old Reflectometry GUI

**Code Review**
- Make sure that there is no more addition to the Wavelength min and max that are retrieved from the IDFs of the reflectometry instruments.
- Make sure the comments that mentioned the addition of constants are now removed
- Make sure all other code changes make sense and are clean.

**Tester** 
- Make sure that your managed data directory is able to use the data archive
- Open the reflectometry interface `Interfaces > Reflectometry > ISIS Reflectometry`
- Change the processing instrument to `SURF`
- Insert this information into the processing table:
- `Run(s):` 114150, `trans1:`114147
- Click `Process`
- Ensure that the `qmin_1` entry is populated with a value close to 0.048
- Ensure that the `qmax_1` entry is populated with a value close to 0.7
- Ensure that the `angle 1` entry is populated with a value of 1.5

**Release Notes** [here](http://www.mantidproject.org/Release_Notes_3_7_Reflectometry#ISIS_Reflectometry)

Fixes #15131
